test/common/EdgeModule.cs

```diff
@@ -0,0 +1,114 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+namespace common
+{
+    using System;
+    using System.Linq;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Devices.Edge.Util;
+
+    public enum EdgeModuleStatus
+    {
+        Running,
+        Stopped
+    }
+
+    public class EdgeModule
+    {
+        protected string deviceId;
+        protected IotHub iotHub;
+
+        public string Id { get; }
+
+        public EdgeModule(string id, string deviceId, IotHub iotHub)
```

Comment:
Edge module should always belong to an Edge device; therefore I think EdgeDevice should be one of the argument in EdgeModule constructor.

Reply:
Conceptually you're right, but practically it doesn't make sense to compose these special-purpose test classes in that way. I originally did it that way, but then simplified it because the only piece of information EdgeModule needs about the device (right now) is its ID.

Reply:
Got it.